tests/visualization_tests/matplotlib_tests/test_pareto_front.py

```diff
@@ -49,7 +49,7 @@ def _check_data(figure: Axes, axis: str, expected: Sequence[int]) -> None:
             )
         )
     )
-    numpy.ma.testutils.assert_array_equal(actual, expected)
+    numpy.ma.testutils.assert_array_equal(actual, expected)  # type: ignore[no-untyped-call]
```

Comment:
I suppose we don't need [`numpy.ma`](https://numpy.org/doc/stable/reference/maskedarray.generic.html) module, which is for masked array.

How about replacing the line with 
```suggestion
    numpy.testing.assert_array_equal(actual, expected)
```

Reply:
Thank you for your suggestion!
I confirmed that addresses the issue.